defaults-next-root6.sh

```diff
@@ -17,8 +17,7 @@ overrides:
       which gfortran || { echo "gfortran missing"; exit 1; }
       which cc && test -f $(dirname $(which cc))/c++ && printf "#define GCCVER ((__GNUC__ << 16)+(__GNUC_MINOR__ << 8)+(__GNUC_PATCHLEVEL__))\n#if (GCCVER < 0x070300)\n#error \"System's GCC cannot be used: we need at least GCC 7.X. We are going to compile our own version.\"\n#endif\n" | cc -xc++ - -c -o /dev/null
   ROOT:
-    version: "v6-14-02+git_%(short_hash)s"
-    tag: "a8cc688c984fec81a3deeacdcec020c0aa93880c"
+    tag: "v6-04-14"
```

Comment:
I just stumbled upon this on accident, but I think this is a typo - I think you want "6-14-04"

Reply:
Well... yes. Let me fix it.